fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1812,6 +1812,7 @@ aggregationFunction
     | MAX '(' DISTINCT? expression ')'
     | MIN '(' DISTINCT? expression ')'
     | SUM '(' DISTINCT? expression ')'
+    | ARRAY_AGG '(' expression (ORDER BY sortItem (',' sortItem)*)? ')'
```

Comment:
`group_concat` will also be like ARRAY_AGG. other aggregation does not support `ORDER BY`, and the parse will return a error msg, otherwise, we should do that.